### PR TITLE
[jwplayer] Add getCues, setCues methods and events

### PR DESCRIPTION
**Pull request by martin-badin**
Discussions: 
#63409
#63510
#56870

Docs: 
https://github.com/jwplayer/jwplayer/blob/master/src/js/api/api.js
https://github.com/jwplayer/jwplayer/blob/master/src/js/view/controls/controls.js

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test jwplayer`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x]  Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/jwplayer/jwplayer-tests.ts [hunk 1]
```diff
@@ -1,2 +1,46 @@
-jwplayer('id').resize(40, 40);
```
Comment (esetnik) [inline types/jwplayer/jwplayer-tests.ts:1]: Why are the resize test cases deleted?

**Review by esetnik — CHANGES_REQUESTED** [review]

File: types/jwplayer/jwplayer-tests.ts (hunk 1 above)
Comment (martin-badin) [reply types/jwplayer/jwplayer-tests.ts:1]: Not deleted. Just moved to other functions that return JWPlayer

Comment (esetnik) [reply types/jwplayer/jwplayer-tests.ts:1]: @martin-badin my bad. I totally missed that 😁

**Review by esetnik — APPROVED** [review]

**Comment by martin-badin** [comment]
Ready to merge

**Pull request merged**